tests/unit/client.py

```diff
@@ -236,6 +240,21 @@ def populate_server():
                 )
 
 
+# Avoid race condition while creating, deleting networks
+def avoid_network_race_condition():
```

Comment:
1. Can't we do this using the SQLAlchemy APIs? I'd prefer to limit the number of ways we write code to access a db.
2. Once we have a show_networkingaction or such in the API, we should probably replace this with calls to that; would you put a note in to that effect?